frontend/appflowy_flutter/lib/plugins/document/presentation/share/share_button.dart

```diff
@@ -94,20 +94,14 @@ class ShareActionList extends StatelessWidget {
       onSelected: (action, controller) async {
         switch (action.inner) {
           case ShareAction.markdown:
-            final exportPath = await FilePicker.platform.saveFile(
+            final exportPath = await getIt<FilePickerService>().saveFile(
               dialogTitle: '',
               fileName: '${view.name}.md',
             );
             if (exportPath != null) {
               docShareBloc.add(DocShareEvent.shareMarkdown(exportPath));
-              showMessageToast('Exported to: $exportPath');
             }
             break;
-          // case ShareAction.copyLink:
-          //   NavigatorAlertDialog(
-          //     title: LocaleKeys.shareAction_workInProgress.tr(),
-          //   ).show(context);
-          //   break;
```

Comment:
We don't really need these comments related to copyLink right?

Reply:
Yes, there are no current plans for it.